packages/plugin-ext-vscode/src/browser/plugin-vscode-commands-contribution.ts

```diff
@@ -19,18 +19,26 @@ import { CommandContribution, CommandRegistry, Command } from '@theia/core';
 import { CommandService } from '@theia/core/lib/common/command';
 import TheiaURI from '@theia/core/lib/common/uri';
 import URI from 'vscode-uri';
+import { ContextKeyService } from '@theia/plugin-ext/lib/main/browser/context-key/context-key';
 
 export namespace VscodeCommands {
     export const OPEN: Command = {
         id: 'vscode.open',
         label: 'VSCode open link'
     };
+
+    export const SET_CONTEXT: Command = {
+        id: 'setContext',
+        label: 'Set context'
```

Comment:
do we need it in the command palette?

Reply:
I've removed `label` attribute, AFAIK it removes command from command palette, right?